python/tk_houdini/base_hooks/node_handler_base.py

```diff
@@ -38,13 +39,17 @@ def __new__(cls, *args, **kwargs):
 
         Prevents us needing to define, in code, shotgun templates and settings.
         """
-        if cls.NODE_TYPE:
+        if cls.NODE_TYPE and cls.NODE_CATEGORY:
             if hasattr(cls, "_work_template"):
                 return super(NodeHandlerBase, cls).__new__(cls, *args, **kwargs)
             engine = sgtk.platform.current_engine()
             node_handlers = engine.get_setting("node_handlers")
             for handler in node_handlers:
-                if handler["node_type"] == cls.NODE_TYPE:
+                conditions = [
+                    handler["node_type"] == cls.NODE_TYPE,
+                    handler["node_category"] == cls.NODE_CATEGORY
+                ]
+                if all(conditions) :
```

Comment:
```suggestion
                matching_node_category_type = (
                    handler["node_type"] == cls.NODE_TYPE
                    and handler["node_category"] == cls.NODE_CATEGORY
                )
                if matching_node_category_type :
```

Constructing a new list of booleans seems excessive